src/commands/analyze/index.ts

```diff
@@ -500,25 +507,19 @@ export default class Analyze extends Command {
 
       switch (this.environment) {
         case Environment.localhost:
-          operatorJobHash = this.networkMonitor.decodeCrossChainMessageSentEvent(
-            receipt,
-            this.networkMonitor.operatorAddress,
-          )
+          operatorJobHash = decodeCrossChainMessageSentEvent(receipt, this.networkMonitor.operatorAddress)
           if (operatorJobHash !== undefined) {
-            args = this.networkMonitor.decodeLzEvent(receipt, this.networkMonitor.lzEndpointAddress[network])
+            args = decodeLzEvent(receipt, this.networkMonitor.lzEndpointAddress[network])
```

Comment:
@alexanderattar nice! @BravoNatalie did a version of this in the index-processor.